Core/Utilities/ElasticSearchDB.py

```diff
@@ -334,20 +334,20 @@ def getUniqueValue( self, indexName, key, orderBy = False ):
       query.aggs.bucket( key,
                          'terms',
                          field = key,
-                         size = 1,
+                         size = 10000,
```

Comment:
I hope I do not have to change very often. Constant in python make no sense. Using a variable is very dangerous as I can change the value any time..., but anyway as you prefer.